library/src/main/java/dev/hichamboushaba/suspendactivityresult/ActivityResultManagerImpl.kt

```diff
@@ -48,7 +48,7 @@ internal object ActivityResultManagerImpl : ActivityResultManager {
                         }
 
                         if (!isLaunched) {
-                            launcher!!.launch(input)
+                            launcher?.launch(input)
```

Comment:
I think I wrote a comment about this, but for some reason it isn't not there 😕 

Please revert this change, I prefer a crash instead of having a non-expected state ([offensive programming](https://en.wikipedia.org/wiki/Offensive_programming#:~:text=Offensive%20programming%20is%20a%20name,are%20not%20fundamentally%20in%20conflict.))

Reply:
https://github.com/hichamboushaba/SuspendActivityResult/pull/1/commits/36e8f06d90bc71db8a9d529391b075df6e430a84